templates/docker-compose/setup.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 set -eu -o pipefail
 
-usage="Usage (Linux): curl -LsS https://raw.githubusercontent.com/supportpal/helpdesk-install/master/templates/docker-compose/setup.sh | bash
+usage="Usage (Linux / MacOS): curl -LsS https://raw.githubusercontent.com/supportpal/helpdesk-install/master/templates/docker-compose/setup.sh | bash
```

Comment:
```suggestion
usage="Usage (Linux / MacOS): bash <(curl -LsS https://raw.githubusercontent.com/supportpal/helpdesk-install/master/templates/docker-compose/setup.sh)
```

This line should match https://docs.supportpal.com/current/Deploy+on+Docker

Does it still work on MacOS?

Reply:
Yes, I've been using the command from the guide.